docs/for_users/execution_environment.md

```diff
@@ -36,4 +36,25 @@ Geoclimate can, among other things, be executed using two tools presented below:
 To see Geoclimate in action, you can consult the two following step-by-step documentations:
 
 - With [OSM](./execution_examples/run_osm.md)
-- With [BD Topo v2](./execution_examples/run_bd_topo_v2.md)
\ No newline at end of file
+- With [BD Topo v2](./execution_examples/run_bd_topo_v2.md)
+
+
+
+## Geoclimate and DB manager
+
+Geoclimate uses a spatial database to store and process in a SQL way the data. 
```

Comment:
Already described in implementation

Reply:
Ah ok. I was searching a place before removing it from the README. So I will remove it